fsharp-backend/src/LibExecution/StdLib/LibString.fs

```diff
@@ -883,8 +883,25 @@ let fns : List<BuiltInFn> =
       fn =
         (function
         | _, [ DStr s; DInt n ] ->
-            let characterCount = int n
-            Value(DStr(s.[characterCount..]))
+            let drop_first_n s (num_egcs : bigint) =
```

Comment:
This looks a bit complex. I think you could have the same effect by doing:

```
s
|> String.toEgcSeq
|> Seq.toList
|> List.skip count
|> String.concat ""

Reply:
Hi @pbiggar, I know there were simpler ways to make some functions work but what I did was to translate the current implementation in Ocalm, like in this one uses Unicode_string.drop_first_, I thought I should be doing that 😅
![image](https://user-images.githubusercontent.com/13671857/103563681-04e09f00-4e72-11eb-815d-97a56b95d6e4.png)